### ISSUE TITLE
Dystonia classification

### ISSUE THREAD

**Issue by cmungall** [obophenotype/human-phenotype-ontology#3672]
```
    is_a HP:0001332 ! Dystonia *** 
       is_a HP:0001304 ! Torsion dystonia
       is_a HP:0002268 ! Paroxysmal dystonia
       is_a HP:0002451 ! Limb dystonia
       is_a HP:0002530 ! Axial dystonia
       is_a HP:0004373 ! Focal dystonia
       is_a HP:0007325 ! Generalized dystonia
       is_a HP:0010553 ! Oculogyric crisis
```

is it not the case that Oculogyric crisis is a subtype of Focal dystonia, rather than a sib?

General Q: we have dystonia classification in MONDO too - should we collapse these to 'dystonia' and defer to HP for the subtyping at the phenotype level?

See also ORDO: https://github.com/Orphanet/ORDO/issues/12

cc @nicolevasilevsky

**Comment by nicolevasilevsky** [comment]
I agree, Oculogyric crisis seems like it should be a subclass of Focal dystonia (that is how it is classified in MONDO). 

I like the idea of having the dystonia subtypes in HP instead of MONDO.
